bazel/repositories.bzl

```diff
@@ -379,6 +383,33 @@ def _net_zlib():
 def _com_google_cel_cpp():
     _repository_impl("com_google_cel_cpp")
 
+    # Parser dependencies
+    http_archive(
+        name = "rules_antlr",
```

Comment:
these should be in repository_locations.bzl

Reply:
Done